_includes/helpers.js

```diff
@@ -0,0 +1,141 @@
+	var table = document.createElement("table");
+	insideNode.appendChild(table);
+	var thead = document.createElement("thead");
+	table.appendChild(thead);
+	var headRow = document.createElement("tr");
+	thead.appendChild(headRow);
+	
+	var nameHead = document.createElement("th");
+	nameHead.appendChild(document.createTextNode("Name"));
+	headRow.appendChild(nameHead);
+	var dateHead = document.createElement("th");
+	dateHead.appendChild(document.createTextNode("Date"));
+	headRow.appendChild(dateHead);
+	var placeHead = document.createElement("th");
+	placeHead.appendChild(document.createTextNode("Place"));
+	headRow.appendChild(placeHead);
+
+	if (includeCFP === true) {
+		var cfpHead = document.createElement("th");
+		cfpHead.appendChild(document.createTextNode("CFP Deadline"));
+		headRow.appendChild(cfpHead);
+	}
+	
+	var tbody = document.createElement("tbody");
+	table.appendChild(tbody);
+	
+	for (var i = 0; i < confs.length; i++) {
+		tbody.appendChild(buildRow(confs[i], includeCFP));
+	}
+
+}
+
+var months = ["", "January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+
+function buildRow(conference, includeCFP) {
+	var tr = document.createElement("tr");
+	
+	var nameCell = document.createElement("td");
+	var conferenceName = document.createTextNode(conference.name);
+	if (conference.link !== null) {
+		var linkNode = document.createElement("a");
+		linkNode.setAttribute("href", conference.link);
+		linkNode.setAttribute("title", conference.name);
+		linkNode.appendChild(conferenceName);
+		nameCell.appendChild(linkNode);
+	} else {
+		nameCell.appendChild(conferenceName);
+	}
+	tr.appendChild(nameCell);
+	
+	var dateNode = document.createElement("td");
+	var start = conference.start;
+	var end = conference.end;
+	var dateString = "";
+	if (start.year === end.year && start.month === end.month && start.day === end.day) {
+		// 1-day conference
+		dateString = months[start.month] + " " + start.day + ", " + start.year;
+	} else  if (start.month !== end.month) {
+		// conference spans the end of a month
+		var startString = months[start.month] + " " + start.day;
+		var endString = months[end.month] + " " + end.day + ", " + end.year;
+		dateString = startString + " – " + endString;
+	} else {
+		// multi-day conference, all within a single month
+		dateString = months[start.month] + " " + start.day + "–" + end.day + ", " + end.year;
+		
+	}
+	var strong = document.createElement("strong");
+	strong.appendChild(document.createTextNode(dateString));
+	dateNode.appendChild(strong);
+	tr.appendChild(dateNode);
+	
+	var placeNode = document.createElement("td");
+	if (conference.location === null || conference.location.length == 0) {
+		placeNode.appendChild(document.createTextNode("TBA"));
+	} else {
+		placeNode.appendChild(document.createTextNode(conference.location));
+	}
+	tr.appendChild(placeNode);
+	
+	if (includeCFP === true) {
```

Comment:
any reason for the verbosity ? Does javascript fail here without the triple-equal ? 
```suggestion
	if (includeCFP) {
```

Reply:
I had some major problems during development with JS truthiness. Using `===` seemed to be the suggested way of dealing with that.